compiler/src/dotty/tools/dotc/transform/sjs/PrepJSInterop.scala

```diff
@@ -814,7 +825,28 @@ class PrepJSInterop extends MacroTransform with IdentityDenotTransformer { thisP
       super.transform(tree)
     }
 
+    /** Removes annotations from exported definitions (e.g. `export foo.bar`):
+     *  - `js.native`
+     *  - `js.annotation.*`
+     */
+    private def stripJSAnnotsOnExported(sym: Symbol)(using Context): Unit =
+      if !sym.is(Exported) then return
```

Comment:
When the branch(es) of an if have side-effects, in the Scala.js coding style we impose them to be on a separate line, so that they stand out:
```suggestion
      if !sym.is(Exported) then
        return
```